test/controllers/callback_controller_test.rb

```diff
@@ -47,6 +54,7 @@ class CallbackControllerTest < ActionController::TestCase
         "AppleWebKit/537.36 (KHTML, like Gecko)"\
         "Chrome/69.0.3497.100 Safari/537.36"
       ShopifyApp::SessionRepository.stubs(:store_session)
+      ShopifyApp.configuration.custom_post_authenticate_tasks = nil
```

Comment:
Should this be the global test config value, so that other tests start with this as an assumption?

Reply:
Not necessary I only did it to "reset" the configuration state, but I can move that to the teardown so it doesn't become and assumption for other tests!